Frontend/src/data/rental/onSave.js

```diff
@@ -29,10 +29,14 @@ const newItemStatus = (rental) => {
   }
 };
 
-const updateItemStatus = async (item, status) => {
+const updateItemStatus = async (item, status, rental) => {
   item.status = status;
   await Database.updateDoc(item);
+  // only insert this attribute after database entry
+  // so that it will not be saved in the database
+  item.rental = rental;
```

Comment:
I don't like this so much. Couldn't we just move the logic for calculating the expected return date in this file and then just pass the date to the WoocommerceClient?

Reply:
Or we could actually add a new attribute expectedReturnDate to Item and also store it in the db. I think this would be nicer and we could also use it somewhere else.